fenix/app/src/main/res/values/strings.xml

```diff
@@ -273,9 +273,15 @@
     <!-- Enable notification pre permission dialog title
         The first parameter is the name of the app defined in app_name (for example: Fenix) -->
     <string name="onboarding_home_enable_notifications_title">Notifications help you do more with %s</string>
+    <!-- Enable notification pre permission dialog title used by Nimbus. Nimbus does not support string placeholders.
+        Note: The word "Firefox" should NOT be translated -->
+    <string name="onboarding_home_enable_notifications_title_nimbus" tools:ignore="UnusedResources">Notifications help you do more with Firefox</string>
```

Comment:
I think the key shouldn't include "home" as this isn't on Home and the strings can be in the onboarding section wiith the rest of the strings. The one above this is for a dialog on Home.